### ISSUE TITLE
Improve page loading User Experience

### ISSUE THREAD

**Issue by tisto** [plone/volto#654]
Volto currently delivers the unstyled server-side rendered version of the HTML and displays that first. This sometimes shows up for a few seconds.

<img width="1387" alt="screenshot 2019-02-27 at 16 57 20" src="https://user-images.githubusercontent.com/632105/53503764-03ddca00-3ab1-11e9-820d-201a9cd0a4e9.png">

[volto-loading.mov.zip](https://github.com/plone/volto/files/2911103/volto-loading.mov.zip)

I think it would significantly improve the user experience if we could add a css animation here. Though, I don't know enought about Razzle to understand where I would have to add such an animation.

@sneridagh @nileshgulia1 ?

**Comment by sneridagh** [comment]
@tisto This behavior is “normal” for the dev server, since the SSR does not have the CSS on it (by design). The style-loader fires after JS is loaded. It doesn’t happen in the production server since the CSS is extracted. In dev mode can’t be like this for live reloading.

**Comment by tisto** [comment]
@sneridagh right. Then we can keep things as they are. I am just wondering where we could hook in css page transition animations. The page transition between pages is standard react router, that shouldn't be a problem. Is there a way to hook into the first page loading via razzle?

**Comment by nzambello** [comment]
I pushed my branch `router-animations` in which I tested some components. The way is the good one and probably not much far from a good result, but this is not enough.
There are animations but laggy and aren't smooth at all.

I'd need some help or suggestions.
